src/components/DashboardCard.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class DashboardCard extends Component<{ cardImage: any; cardURL: string }> {
```

Comment:
https://stackoverflow.com/questions/41791933/in-typescript-what-is-the-type-of-image

cardImage should be of type HTMLImageElement I believe

Reply:
Looks like importing the image as I did assigns the image path to the cardImage object, and hence weirdly it's actually a string and not an image.